protocol/communities_messenger_test.go

```diff
@@ -618,15 +620,13 @@ func (s *MessengerCommunitiesSuite) TestImportCommunity() {
 		if len(response.Communities()) == 0 {
 			return errors.New("community not received")
 		}
+		if !response.Communities()[0].IsAdmin() {
+			return errors.New("isn't admin despite import")
+		}
```

Comment:
I tried putting the `Joined()` check for alice in here as well, but it always makes the test fail in a weird way.

Reply:
looking into it, would you mind pushing the failed test if you have time please?

Reply:
It's already pushed.

The tests that are failing are `TestRequestAccess` and `TestRequestAccessAgain`

Let me know if I can be of any assistance and thanks for taking the time!